tpcx_bb/benchmark_runner/draco/client.sh

```diff
@@ -0,0 +1,14 @@
+bash /root/tpcx-bb/tpcx_bb/cluster_configuration/cluster-startup.sh SCHEDULER &
+
+CONDA_ENV_NAME="rapids-tpcx-bb"
+CONDA_ENV_PATH="/conda/etc/profile.d/conda.sh"
+source $CONDA_ENV_PATH
+conda activate $CONDA_ENV_NAME
+ 
+echo "STARTED SCHEDULER"
+sleep 10
```

Comment:
Need to add an equivalent of the old `dask-waiter.py` that sleeps until number of expected workers are present. Allocations can take awhile to be granted!

Reply:
That would be nice, but I don't think it needs to block this PR necessarily. Users can simply manually wait.